packages/cli-repl/test/fixtures/load/hello1.js

```diff
@@ -1 +1,2 @@
+/* eslint-disable no-undef */
```

Comment:
Probably can add all fixtures in eslintignore? This also works though, so feel free to ignore!

Reply:
I think we should probably do that if it gets out of control but in the meantime it is nice in that we still get some useful checks.

Reply:
Good point!